.travis.yml

```diff
@@ -2,10 +2,18 @@ language: c++
 sudo: false
 env:
   - NODE_VERSION="0.10"
+  - NODE_VERSION="0.11"
   - NODE_VERSION="0.12"
+  - NODE_VERSION="iojs-v1"
```

Comment:
If iojs is merged back into the project, shouldn't it be ok to drop support for it? Is iojs actually still supported, or was it dropped in favor of node 4?

From #307:

> 0.11 is an unstable version that is not supported or updated anymore. 0.12 is fully compatible with 0.11 and as it is in maintenance mode, no breaking changes will be committed there.

Since it just slows CI down, and I don't see a strong benefit, I'd vote against including these versions.